resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/features/BasicDriverFeatureStep.scala

```diff
@@ -44,6 +44,11 @@ private[spark] class BasicDriverFeatureStep(conf: KubernetesDriverConf)
 
   // CPU settings
   private val driverCpuCores = conf.get(DRIVER_CORES.key, "1")
+  private val driverCoresRequest = if (conf.contains(KUBERNETES_DRIVER_REQUEST_CORES)) {
+    conf.get(KUBERNETES_DRIVER_REQUEST_CORES).get
+  } else {
+    driverCpuCores
+  }
```

Comment:
Thank you for making a PR, @arunmahadevan . Could you rewrite like the following one-liner?
```scala
-  private val driverCoresRequest = if (conf.contains(KUBERNETES_DRIVER_REQUEST_CORES)) {
-    conf.get(KUBERNETES_DRIVER_REQUEST_CORES).get
-  } else {
-    driverCpuCores
-  }
+  private val driverCoresRequest = conf.get(KUBERNETES_DRIVER_REQUEST_CORES.key, driverCpuCores)
```

Reply:
Thanks for the suggestion. I just followed the pattern used somewhere else in the code.